src/sys/timerfd.rs

```diff
@@ -0,0 +1,124 @@
+//! Timer API via file descriptors.
+//!
+//! Timer FD is a Linux-only API to create timers and get expiration notifications through file
+//! descriptors.
+//!
+//! For more documentation, please read [timerfd_create(2)](http://man7.org/linux/man-pages/man2/timerfd_create.2.html).
+//!
+//! # Examples
+//!
+//! Create a new one-shot timer that expires after 5 seconds.
+//! ```no_run
```

Comment:
Why the `no_run`?  You might as well run it, though maybe decrease the timer to 1 second.  Also, the example would be more useful if there were a `read` at the end.

Reply:
Ok I removed the no_run, decreased the timer and added a read at the end.